Sources/BraintreeCard/CreditCardGraphQLBody.swift

```diff
@@ -0,0 +1,178 @@
+// swiftlint:disable nesting
+
+import Foundation
+
+struct CreditCardGraphQLBody: Encodable {
```

Comment:
Can we add a docstring for this? Can use `/// The POST body for <endpoint_here>` like we have in other files.